microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -102,6 +112,36 @@ def verify_gpu_provision(self, node: Node, log: Logger) -> None:
     def verify_max_gpu_provision(self, node: Node, log: Logger) -> None:
         _gpu_provision_check(8, node, log)
 
+    @TestCaseMetadata(
+        description="""
+            This test case verifies if gpu drivers are installed using extension.
+
+            Steps:
+            1. Install the GPU Driver using Extension.
+            2. Reboot and check for kernel panic
+            3. Validate gpu drivers can be loaded successfully.
+
+        """,
+        timeout=TIMEOUT,
+        requirement=simple_requirement(
+            supported_features=[GpuEnabled(), SerialConsole, AzureExtension],
+        ),
+        priority=2,
+    )
+    def verify_gpu_extension_installation(
+        self, node: Node, log_path: Path, log: Logger
+    ) -> None:
+        gpu_feature = node.features[Gpu]
+        try:
+            gpu_feature._install_driver_using_platform_feature()
+        except LisaException as e:
+            if "GPUExtensionNotSupported" in str(e):
+                raise SkippedException("GPU Extension Installation is not supported")
+            raise e
```

Comment:
> Give a more specific error message. The Azure GPU extension doesn't support this distro version. The "NotSupported" here is not clear enough.

@squirrelsc , "GPUExtensionNotSupported" is not directly raised to the user. It is caught and reraised here

Reply:
If so, it should use a type of exception, not a string. It's more reliable. And the message is not clear enough too.